models/experimental/functional_unet/tt/unet_shallow_ttnn.py

```diff
@@ -322,8 +339,13 @@ def __init__(
                 output_channels=self.conv1.out_channels,
                 compute_grid_size=device.compute_with_storage_grid_size(),
                 block_shard_orientation=ttnn.ShardOrientation.ROW_MAJOR,
+                enable_channels_padding=True,
```

Comment:
What does this do? Is it required for correctness or just performance/L1 usage?

Reply:
If you ask about enable_channels_padding is the param in the function and defaults to true for conv2d (this method is used for max_pool2d as well)

Reply:
In this case enable_channels_padding doesn't have an impact since HEIGHT_SHARDED is being used and it only affects block and width sharding